clients/venice-admin-tool/src/main/java/com/linkedin/venice/ZkCopier.java

```diff
@@ -0,0 +1,196 @@
+package com.linkedin.venice;
+
+import static com.linkedin.venice.VeniceZkPaths.ADMIN_TOPIC_METADATA;
+import static com.linkedin.venice.VeniceZkPaths.EXECUTION_IDS;
+import static com.linkedin.venice.VeniceZkPaths.PARENT_OFFLINE_PUSHES;
+import static com.linkedin.venice.VeniceZkPaths.ROUTERS;
+import static com.linkedin.venice.VeniceZkPaths.STORES;
+import static com.linkedin.venice.VeniceZkPaths.STORE_GRAVEYARD;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.commons.io.FileUtils;
+
+
+public class ZkCopier {
+  private static final Set<String> VENICE_ZK_PATHS = new HashSet<>(
```

Comment:
After going through the PR, I realized that this only contains the ZK paths in the cluster. Let's rename this to `CLUSTER_ZK_PATHS`